### PR TITLE
Add a note to readme concerning issue #167.

### PR DESCRIPTION
**Pull request by kindaro**
Just a few words concerning placement of setup code, and a modification of the example to make the point noticeable even at a slightest glance.

This is in accordance with the discussion in the issue #167.

### CODE REVIEW

**Comment by kindaro** [comment]
It fails the check due to extra space character in one of the tests' output:

```
  test/PropertySpec.hs:51: (best-effort)
  1) Property.runProperty reports the value for which a property fails
       expected: Failure "*** Failed! Falsifiable (after 1 test): \n0"
        but got: Failure "*** Failed! Falsifiable (after 1 test):\n0"
```

I have no idea why. The same test is consistently passed on my machine. But, this commit `438ec5a5e551e3f02581fd13064aef9f7d5e0bdc` (`Re #176: Bumped development version to 0.14.0.`) also has failed checks, and for the same reason. I don't know what to make out of it.

**Comment by andreasabel** [comment]
Seems like something changed in QuickCheck.

**Comment by kindaro** [comment]
What should I do? I don't think the issue with this one unrelated failing test should be addressed in this pull request, or should it?

File: README.markdown [hunk 1]
```diff
@@ -124,6 +127,12 @@ foo :: Int
 foo = 42
 ```
 
+Note that you should not place setup code inbetween the module header (`module
```
Comment (andreasabel) [inline README.markdown]: Could the reason be that in Haskell `import` statements have to be on top of the file?  Have you tried putting setup code here that only imports a file?
(Just my wild guessing.)

**Comment by andreasabel** [comment]
Yes, certainly the travis failure should be fixed in a different PR.

File: README.markdown (hunk 1 above)
Comment (kindaro) [reply README.markdown]: I can't understand what you mean by "putting setup code here that only imports a file"?

Comment (andreasabel) [reply README.markdown]: Sorry, that was a wrong guess on my side.

**Pull request merged**